### ISSUE TITLE
BioMixer missing top menu bar

### ISSUE THREAD

**Issue by jvendetti** [ncbo/virtual_appliance#6]
End user reported that the BioMixer visualization software is missing the top menu bar in their virtual appliance installation. I was able to reproduce this on [Biblioportal](http://biblio.ontoportal.org/):

1. Go to Bibliportal and navigate to any ontology, e.g., [BIBFRAME](http://biblio.ontoportal.org/ontologies/BIBFRAME/?p=summary).
2. Open the Classes tab and select any class in the tree.
3. Click the Visualization tab on the right-hand side.

Notice that the top menu bar is missing most of the content, e.g., "Fullscreen", etc.

<img width="865" alt="Screenshot 2019-04-04 13 46 37" src="https://user-images.githubusercontent.com/1696923/55587400-21213c00-56e0-11e9-8e40-27e6547c3906.png">

This is the expected appearance, as seen on production BioPortal:

<img width="763" alt="Screenshot 2019-04-04 13 48 39" src="https://user-images.githubusercontent.com/1696923/55587520-6180ba00-56e0-11e9-8bc5-e8076ee4410a.png">

Chrome developer console shows the following 500 error after clicking on any class:

```
POST http://biblio.ontoportal.org/BioMixer/org.thechiselgroup.biomixer.BioMixerWorkbench/authentication 500 (Internal Server Error)
```

**Comment by graybeal** [comment]
@alexskr I notice that the VA REST service seems to be at :8080, unlike the production REST API; and the BioMixer address in the error messages in #7 seem to include :8080 (but this error message does not include :8080 in the BioMixer REST address).  Is there any chance that's related?

**Comment by alexskr** [comment]
this issue is resolved in v3.0 of the appliance
